src/components/interface/PendingTransaction/PendingTransaction.tsx

```diff
@@ -62,7 +62,7 @@ const PendingTransaction: React.FunctionComponent<PendingTransactionProps> = ({
   const cachedMargin = useRef<number | undefined>(margin);
 
   useEffect(() => {
-    if (!isUndefined(margin) && margin !== 0){
+    if (!isUndefined(margin) && margin !== 0 && fetch == 0){
```

Comment:
use === instead of ==. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#:~:text=Triple%20equals%20(%20%3D%3D%3D%20)%20will%20do,types%20differ%2C%20false%20is%20returned.

Reply:
resolved in [00d1ca0](https://github.com/Voltz-Protocol/voltz-ui/pull/262/commits/00d1ca048469411bbc9464c9b09cbf4ba0d11668)